mods/ra/maps/soviet-05/main.lua

```diff
@@ -0,0 +1,223 @@
+IdleHunt = function(unit) if not unit.IsDead and not unit.Type == harv then Trigger.OnIdle(unit, unit.Hunt) end end
```

Comment:
Nope, I don't want harvesters to fall into Hunt() to get stuck. It shouldn't happen but I did it to be sure it's not cause of the harvester issues